scripts/bake-dependencies.sh

```
@@ -0,0 +1,186 @@
+# - WebKit (JavaScriptCore)
+# - BoringSSL
+# - And other cmake-managed dependencies
+
+set -eu
+
+print() {
+    echo "$@"
+}
+
+error() {
+    print "error: $@" >&2
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)